test/src/test/java/lib/form/TextAreaTest.java

```diff
@@ -72,7 +72,7 @@ public FormValidation doCheckText2(@QueryParameter String text1) {
     @Issue("JENKINS-27505")
     @Test
     public void text() throws Exception {
-        T1: {
+        {
```

Comment:
A proper refactoring would be to split it to separate tests. 
No string opinion about deletion, we have comments anyway